Cesium.CodeGen/Generators/Functions.cs

```diff
@@ -46,6 +46,7 @@ private static (MethodDefinition, bool isMain) GenerateMethod(TranslationUnitCon
                 scope.Parameters.Add(parameter.Name, parameter);
         }
 
+        function = new FunctionDefinition(function.Specifiers, function.Declarator, function.Declarations, Lowering.LowerStatement(function.Statement));
```

Comment:
_(Minor)_ 
```suggestion
        function = function with { Statement = Lowering.LowerStatement(function.Statement) };
```

Reply:
Nice. I cannot remember that syntax without internet. VSCode was not helping